yalla-coop/website-redesign
# Setup Commitlint

**Issue**: #30 

## What did I do?
Installed the commitlint package and configured it as mentioned in the issue above
- [x] Install and configure commitlint 
- [x] Add a pre-commit to lint files

## How to test?
Make some changes on any file and then try to write a normal commit, the commitlint should yell at you and some linting should happen before that
